jelares/portfolio/src/main/java/com/google/sps/servletData/BlogPost.java

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servletData;
+
+/** Represents a blog post */
+public final class BlogPost {
+  private final String imageUrl;
+  private final String header;
+  private final String content;
+
+  public BlogPost(String imageUrl, String header, String content) {
+    this.imageUrl = imageUrl;
+    this.header = header;
+    this.content = content;
+  }
+
+  /** Get the ImageUrl */
+  public String getImageUrl() {
+    return imageUrl;
+  }
+
+  /** Get the content */
+  public String getContent() {
+    return content;
+  }
+
+  /** Get the header */
+  public String getHeader() {
```

Comment:
Nit: "header" is a bit ambiguous, since a header can contain various types of content. Do you mean "title", which is more specific to text?

Reply:
Sure, I updated it to say title